src/QirTools/pyqir/pyqir/__init__.py

```diff
@@ -1,59 +1,214 @@
+# Copyright(c) Microsoft Corporation.
+# Licensed under the MIT License.
+
 from .pyqir import *
 
+
 class QirBuilder:
+    """
+    The QirBuilder objct describes quantum circuits and emits QIR
+
+    :param module: name of the QIR module
+    :type module: str
+    """
+
     def __init__(self, module: str):
         self.pyqir = PyQIR(module)
 
-    def cx(self, control:str, target:str):
+    def cx(self, control: str, target: str):
```

Comment:
may i recommend using `x_ctr` or `ctr_x` so it's easy to define controlled versions of all intrinsics?

Reply:
In this case I think the naming is intentional, since only `cx` and `cz` are supported. This matches what we do for some target packages and the Base Profile where only specific controlled gates are supported and not arbitrary controls like in full Q#.

Reply:
The naming was based on the base profile qubit names except I expanded `q` `=>` `qubit`